mystery-deckbuilder/Assets/Scripts/Notepad/Chapter.cs

```diff
@@ -0,0 +1,66 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Chapter 
```

Comment:
NIT: As per practices guide, you should add the description of the class as a multi-line comment at the top. You've done a good job to in-line comment though, so it is not a big deal!